src/models/orderDetails/IOrderDetils.ts

```diff
@@ -0,0 +1,8 @@
+export interface IOrderDetails {
+  id?: number;
+  order_details_id: string;
+  line_items?: [];
```

Comment:
I think we need to indicate what kind of array do we want to specify, for  example any [] object [] or IItem [] etc